Lib/pydoc.py

```diff
@@ -1254,6 +1254,17 @@ def makename(c, m=object.__module__):
                 push('    ' + makename(base))
             push('')
 
+        # List the built-in subclasses, if any:
+        subclasses = [
+            cls.__name__ for cls in object.__subclasses__()
+            if cls.__module__ == 'builtins'
+        ]
+        if subclasses and object.__class__ != type:
```

Comment:
Are you sure you didn't want `object is not builtins.object`? What you have now only shows subclasses for objects with a metaclass.

Reply:
Oh, yes. You're correct! I've updated the patch to reflect the changes.